cloud/deployment/deployment_test.go

```diff
@@ -1219,6 +1227,31 @@ func (s *Suite) TestCreate() {
 		mockPlatformCoreClient.AssertExpectations(s.T())
 	})
 
+	s.Run("success with Astro Executor on Dedicated", func() {
+		// Set up mock responses and expectations
+		mockCoreClient.On("GetDeploymentOptionsWithResponse", mock.Anything, mock.Anything, mock.Anything).Return(&GetDeploymentOptionsResponseOK, nil).Once()
+		mockCoreClient.On("ListWorkspacesWithResponse", mock.Anything, mock.Anything, mock.Anything).Return(&ListWorkspacesResponseOK, nil).Once()
+		mockPlatformCoreClient.On("ListClustersWithResponse", mock.Anything, mock.Anything, mock.Anything).Return(&mockListClustersResponse, nil).Once()
+		mockPlatformCoreClient.On("CreateDeploymentWithResponse", mock.Anything, mock.Anything, mock.Anything).Return(&mockCreateDeploymentResponse, nil).Once()
+		mockPlatformCoreClient.On("GetDeploymentWithResponse", mock.Anything, mock.Anything, mock.Anything).Return(&deploymentResponse, nil).Once()
+
+		// Mock user input for deployment name and wait for status
+		defer testUtil.MockUserInput(s.T(), "test-name")()
+		defer testUtil.MockUserInput(s.T(), "y")()
+
+		// setup wait for test
+		sleepTime = 1
+		tickNum = 2
```

Comment:
How are these used?

Reply:
these affect how long we healthy HealthPoll